site/static/main.js

```diff
@@ -1,36 +1,34 @@
 /* tslint:disable:no-unused-variable */
```

Comment:
But you will have to run tsc locally in this folder and etc. (or import from `build/site/static`). IMHO, this complexity already outweighs the minor benefits you like, so I'm *not* gonna do that.  If you prefer that you can submit another PR later.